pygmt/src/triangulate.py

```diff
@@ -0,0 +1,384 @@
+"""
+triangulate - Delaunay triangulation or Voronoi partitioning and gridding of
+Cartesian data.
+"""
+import warnings
+
+import pandas as pd
+from pygmt.clib import Session
+from pygmt.exceptions import GMTInvalidInput
+from pygmt.helpers import (
+    GMTTempFile,
+    build_arg_string,
+    fmt_docstring,
+    kwargs_to_strings,
+    use_alias,
+)
+from pygmt.io import load_dataarray
+
+
+class triangulate:  # pylint: disable=invalid-name
+    """
+    Delaunay triangulation or Voronoi partitioning and gridding of Cartesian
+    data.
+
+    Triangulate reads in x,y[,z] data and performs Delaunay triangulation,
+    i.e., it finds how the points should be connected to give the most
+    equilateral triangulation possible. If a map projection (give *region* and
+    *projection*) is chosen then it is applied before the triangulation is
+    calculated. By default, the output is triplets of point id numbers that
+    make up each triangle. The id numbers refer to the points position (line
+    number, starting at 0 for the first line) in the input file. If **outgrid**
+    and **spacing** are set a grid will be calculated based on the surface
+    defined by the planar triangles. The actual algorithm used in the
+    triangulations is either that of Watson [1982] or Shewchuk [1996] [Default]
+    (if installed; type **gmt get GMT_TRIANGULATE** on the command line to see
+    which method is selected). This choice is made during the GMT installation.
```

Comment:
```suggestion
    triangulations is either that of Watson [1982] or Shewchuk [1996] [Default
    is Shewcuck, 1996 if installed; type **gmt get GMT_TRIANGULATE** on the
    command line to see which method is selected].
```
I would assume one could set `GMT_TRIANGULATE`, in which case 'This choice is made during GMT installation' may only be partially accurate.

Second suggested change relates to using [Default is ...] rather than just [Default] to be more clear.